pkg/config/config.go

```diff
@@ -895,9 +895,9 @@ func DefaultConfig() (*Config, error) {
 		RuntimeConfig: RuntimeConfig{
 			AllowedDevices:     []string{"/dev/fuse", "/dev/net/tun"},
 			DecryptionKeysPath: "/etc/crio/keys/",
-			DefaultRuntime:     defaultRuntime,
+			DefaultRuntime:     DefaultRuntime,
 			Runtimes: Runtimes{
-				defaultRuntime: defaultRuntimeHandler(),
+				DefaultRuntime: defaultRuntimeHandler(),
 			},
```

Comment:
Can't be private any more? How did we miss this last time... :smile:

Reply:
I think having it public is not too bad, isn't it?

Reply:
Can be. :+1: